tests/storage/export.go

```diff
@@ -0,0 +1,449 @@
+			pod.Spec.Containers[0].VolumeDevices = append(pod.Spec.Containers[0].VolumeDevices, k8sv1.VolumeDevice{
+				Name:       volumeName,
+				DevicePath: "/dev/volume",
+			})
+		} else {
+			pod.Spec.Containers[0].VolumeMounts = append(pod.Spec.Containers[0].VolumeMounts, k8sv1.VolumeMount{
+				Name:      volumeName,
+				MountPath: "/data",
+			})
+		}
+		pod.Spec.Containers[0].VolumeMounts = append(pod.Spec.Containers[0].VolumeMounts, k8sv1.VolumeMount{
+			Name:      "cacerts",
+			ReadOnly:  true,
+			MountPath: "/cacerts",
+		})
+		return tests.RunPod(pod)
+	}
+
+	createSourcePodChecker := func(pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
+		volumeName := pvc.GetName()
+		podName := "download-pod"
+		pod := tests.RenderPod(podName, []string{"/bin/sh", "-c", "sleep 360"}, []string{})
+		pod.Spec.Volumes = append(pod.Spec.Volumes, k8sv1.Volume{
+			Name: volumeName,
+			VolumeSource: k8sv1.VolumeSource{
+				PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: pvc.GetName(),
+				},
+			},
+		})
+
+		volumeMode := pvc.Spec.VolumeMode
+		if volumeMode != nil && *volumeMode == k8sv1.PersistentVolumeBlock {
+			pod.Spec.Containers[0].VolumeDevices = append(pod.Spec.Containers[0].VolumeDevices, k8sv1.VolumeDevice{
+				Name:       volumeName,
+				DevicePath: "/dev/volume",
+			})
+		} else {
+			pod.Spec.Containers[0].VolumeMounts = append(pod.Spec.Containers[0].VolumeMounts, k8sv1.VolumeMount{
+				Name:      volumeName,
+				MountPath: "/data",
+			})
+		}
+		return tests.RunPod(pod)
+	}
+
+	createTriggerPodForPvc := func(pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
+		volumeName := pvc.GetName()
+		podName := fmt.Sprintf("bind-%s", volumeName)
+		pod := tests.RenderPod(podName, []string{"/bin/sh", "-c", "sleep 1"}, []string{})
+		pod.Spec.Volumes = append(pod.Spec.Volumes, k8sv1.Volume{
+			Name: volumeName,
+			VolumeSource: k8sv1.VolumeSource{
+				PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: pvc.GetName(),
+				},
+			},
+		})
+
+		volumeMode := pvc.Spec.VolumeMode
+		if volumeMode != nil && *volumeMode == k8sv1.PersistentVolumeBlock {
+			pod.Spec.Containers[0].VolumeDevices = append(pod.Spec.Containers[0].VolumeDevices, k8sv1.VolumeDevice{
+				Name:       volumeName,
+				DevicePath: "/dev/volume",
+			})
+		} else {
+			pod.Spec.Containers[0].VolumeMounts = append(pod.Spec.Containers[0].VolumeMounts, k8sv1.VolumeMount{
+				Name:      volumeName,
+				MountPath: "/data",
+			})
+		}
+		return tests.RunPodAndExpectCompletion(pod)
+	}
+
+	isWaitForFirstConsumer := func(storageClassName string) bool {
+		sc, err := virtClient.StorageV1().StorageClasses().Get(context.Background(), storageClassName, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		return sc.VolumeBindingMode != nil && *sc.VolumeBindingMode == storagev1.VolumeBindingWaitForFirstConsumer
+	}
+
+	ensurePVCBound := func(pvc *k8sv1.PersistentVolumeClaim) {
```

Comment:
No because this function is to ensure the PVC is bound in case of WFFC storage, otherwise we wait for it to get bound (normal provisioner behavior). If WFFC is true, make sure the pvc is pending (it will be without any pods using it) and we create a pod to trigger the binding.